### ISSUE TITLE
UI improvement: Readability with certain colors

### ISSUE THREAD

**Issue by MikeStirner** [albertvaka/bettercounter#56]
Sometimes I use this app in the car and it becomes hard to read the counters that are the color lime green, for example, because the text is white and so there's not a lot of contrast. So I kind of have to squint for a little longer, which makes it less safe when I'm driving. This is because I have to look for an extra second for my eyes to adjust to the darkness of the car compared to the sunny outside.

The only reason I'm trying to read it is to see if I pressed it or not. I'm trying to look for the words "just now".

And the reason it's only a problem when I'm driving is because I'm trying to conserve batteries by keeping my screen kind of at medium brightness. But that makes the text harder to read for those colors.

Some ideas to fix this would be to add some darker colors (like golden brown) or just make those colors darker. e.g. Dark green instead of lime green, etc.

Another would be to make the text black instead of white for the lighter colors.

Or a color picker, so people can choose any color they want for both the background and the text.

Thoughts?

**Comment by albertvaka** [comment]
The colors we have are not so dark, so probably all except the default one should need dark text for contrast. I did a test and it would look like this:

![image](https://github.com/albertvaka/bettercounter/assets/980842/71e8059b-e301-42aa-9b82-e64346e18441)

I don't like the mix of dark and light text that much, though. What do you think? A simpler solution would be to just to make all the colors a bit darker.

If we want to add a color picker, then for sure we will need to allow dark & light text mixed (and detect how dark the background is to select an appropriate text color).

**Comment by MikeStirner** [comment]
Yeah, I think the color picker would be cool, but probably overkill for most people.

And I agree that the mix of white and black text doesn't look very nice. 

So I also agree that just making all the colors darker is the best choice.

**Comment by albertvaka** [comment]
I've published v4.7.0 which changes the colors.

**Comment by MikeStirner** [comment]
I love the darker colors, but they only take effect if I edit the color and choose the new color. Otherwise they stay the bright colors. Not sure if this was the intention, or if it's a bug.
